examples/reference/panes/VTKJS.ipynb

```diff
@@ -0,0 +1,216 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import panel as pn\n",
+    "pn.extension('vtk')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The ``VTK`` pane can renders vtk.js files inside a panel, making it possible to load and interact with complex geometries in 3D.\n",
+    "\n",
+    "#### Parameters:\n",
+    "\n",
+    "For layout and styling related parameters see the [customization user guide](../../user_guide/Customization.ipynb).\n",
+    "\n",
+    "* **``axes``** (dict): A dictionary with the parameters of the axes to construct in the 3d view.\n",
+    "    Must contain at least ``xticker``, ``yticker`` and ``zticker``.\n",
+    "    - ``[x|y|z]ticker`` is a dictionary which contains:\n",
+    "        - ``ticks`` (array of numbers) - required. Positions in the scene coordinates\n",
+    "        of the coresponding axe ticks\n",
+    "        - ``labels`` (array of strings) - optional. Label displayed respectively to\n",
+    "        the `ticks` positions.\n",
+    "\n",
+    "        If `labels` are not defined they are infered from the `ticks` array.\n",
+    "    - ``digits``: number of decimal digits when `ticks` are converted to `labels`.\n",
+    "    - ``fontsize``: size in pts of the ticks labels.\n",
+    "    - ``show_grid``: boolean. If true (default) the axes grid is visible.\n",
+    "    - ``grid_opacity``: float between 0-1. Defines the grid opacity.\n",
+    "    - ``axes_opacity``: float between 0-1. Defines the axes lines opacity.\n",
+    "* **``camera``** (dict): A dictionary reflecting the current state of the VTK camera\n",
+    "* **``enable_keybindings``** (bool): A boolean to activate/deactivate keybindings. Bound keys are:\n",
+    "  - s: set representation of all actors to *surface*\n",
+    "  - w: set representation of all actors to *wireframe*\n",
+    "  - v: set representation of all actors to *vertex*\n",
+    "  - r: center the actors and move the camera so that all actors are visible\n",
+    "  \n",
+    "  The mouse must be over the pane to work\n",
+    "  <br>**Warning**: These keybindings may not work as expected in a notebook context, if they interact with already bound keys\n",
+    "* **``orientation_widget``** (bool): A boolean to activate/deactivate the orientation widget in the 3D pane. This widget is clickable and allows to rotate the scene in one of the orthographic projections.\n",
+    "* **``object``** (str or object): Can be a string pointing to a local or remote file with a `.vtkjs` extension.\n",
+    "___"
```

Comment:
```suggestion
    "The ``VTK`` pane renders vtk.js files inside a panel, making it possible to load and interact with complex geometries in 3D.\n",
    "\n",
    "#### Parameters:\n",
    "\n",
    "For layout and styling related parameters see the [customization user guide](../../user_guide/Customization.ipynb).\n",
    "\n",
    "* **``axes``** (dict): A dictionary with the parameters of the axes to construct in the 3D view.\n",
    "    Must contain at least ``xticker``, ``yticker`` and ``zticker``.\n",
    "    - ``[x|y|z]ticker`` is a dictionary which contains:\n",
    "        - ``ticks`` (array of numbers) - required. Positions in the scene coordinates\n",
    "        of the corresponding axis ticks\n",
    "        - ``labels`` (array of strings) - optional. Label displayed respectively to\n",
    "        the `ticks` positions.\n",
    "\n",
    "        If `labels` are not defined they are inferred from the `ticks` array.\n",
    "    - ``digits``: number of decimal digits when `ticks` are converted to `labels`.\n",
    "    - ``fontsize``: size in pts of the ticks labels.\n",
    "    - ``show_grid``: boolean. If true (default) the axes grid is visible.\n",
    "    - ``grid_opacity``: float between 0-1. Defines the grid opacity.\n",
    "    - ``axes_opacity``: float between 0-1. Defines the axes lines opacity.\n",
    "* **``camera``** (dict): A dictionary reflecting the current state of the VTK camera\n",
    "* **``enable_keybindings``** (bool): A boolean to activate/deactivate keybindings. Bound keys are:\n",
    "  - s: set representation of all actors to *surface*\n",
    "  - w: set representation of all actors to *wireframe*\n",
    "  - v: set representation of all actors to *vertex*\n",
    "  - r: center the actors and move the camera so that all actors are visible\n",
    "  \n",
    "  The mouse must be over the pane to work.\n",
    "  <br>**Warning**: These keybindings may not work as expected in a notebook context, if they interact with already bound keys.\n",
    "* **``orientation_widget``** (bool): A boolean to activate/deactivate the orientation widget in the 3D pane. This widget is clickable and allows to rotate the scene in one of the orthographic projections.\n",
    "* **``object``** (str or object): Can be a string pointing to a local or remote file with a `.vtkjs` extension.\n",
    "___"
```

Not sure what those ending underscores do.

Reply:
the ending underscore are for the horizontal separator;
![image](https://user-images.githubusercontent.com/18531147/81834754-b0cf2600-9541-11ea-98b4-bdbc47776c60.png)

Reply:
Looks nice!